src/views/v2/month/calendar-body/day/multiday-events/date/featured-icon.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * View: Month View - Calendar Event Featured Icon
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/month/calendar-body/day/multiday-events/date/featured-icon.php
```

Comment:
This is fine for now, but as you break the multiday event out in the multiday event ticket, this is going to change. When you change it, change it to `featured.php`